simplesat/constraints/tests/test_package_parser.py

```diff
@@ -226,30 +226,36 @@ def test_complicated(self):
 
         # Given
         package_string = '; '.join((
-            "bokeh 0.2.0-3",
+            "bokeh_git 0.2.0-3",
             "install_requires (zope *, numpy ^= 1.8.0, requests >= 0.2)",
-            "conflicts (requests ^= 0.2.5, requests > 0.4, bokeh_git)",
+            "conflicts (requests ^= 0.2.5, requests > 0.4, bokeh)",
+            "provides (webplot ^= 0.1, bokeh)",
```

Comment:
This is only testing our ability to parse, which separates the clauses and builds constraint tuples, but doesn't validate. In general we don't validate things up front due to the overhead of parsing version objects and the sheer number of packages. Also, we will not be parsing pretty strings during a normal work flow. IThe program which is using `simplesat` as a library will construct package metadata directly (which is why we guard against it below this level).